Trident/offsetfinder.c

```diff
@@ -1,4 +1,10 @@
-//
+uint32_t find_OSSymbol_getMetaClass(void) {
```

Comment:
Where are other offsets? Please just add a case for your device + OS combination